modules/googleplay/src/main/scala/cards/nine/googleplay/service/free/interpreter/cache/CacheInterpreter.scala

```diff
@@ -16,17 +16,16 @@
 package cards.nine.googleplay.service.free.interpreter.cache
 
 import cards.nine.commons.redis._
-import cards.nine.domain.application.Package
-import cards.nine.googleplay.service.free.algebra.Cache._
-import cats.~>
+import cards.nine.domain.application.{ FullCard, Package }
+import cards.nine.googleplay.service.free.algebra.Cache
 import cats.instances.list._
 import cats.syntax.cartesian._
 import cats.syntax.functor._
 import cats.syntax.traverse._
 import org.joda.time.{ DateTime, DateTimeZone }
 import scala.concurrent.ExecutionContext
 
-class CacheInterpreter(implicit ec: ExecutionContext) extends (Ops ~> RedisOps) {
+class CacheInterpreter(implicit ec: ExecutionContext) extends Cache.Handler[RedisOps] {
```

Comment:
IMHO, we should not use implicits. The `implicit` has a readability overhead in that it forces you to remember that something else is being passed. If this happens at many locations, such as in a `for`-`yield`, it is useful. In this case, where we will create one instance used in one place, not so much.